Index: express-bandbase/models/song.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>var mongoose = require('mongoose');\r\nconst {DateTime} = require(\"luxon\");\r\nconst url = require(\"url\");\r\n\r\nvar Schema = mongoose.Schema;\r\n\r\nvar SongSchema = new Schema({\r\n    title: {type: String, required: true, maxLength: 20},\r\n    URL_videoclip: {type: String},\r\n    data_of_release: {type: Date},\r\n    artist: {type: Schema.Types.ObjectId, ref: 'Artist', required: true},\r\n});\r\n\r\n//Virtual for the release\r\nSongSchema\r\n    .virtual('release')\r\n    .get(function () {\r\n        releaseDate = DateTime.fromJSDate()(this.date_of_release).toLocaleString(DateTime.DATE_MED);\r\n        return releaseDate;\r\n    });\r\n\r\n// Virtual for author's URL\r\nSongSchema\r\n    .virtual('url')\r\n    .get(function () {\r\n        return '/discover/song/' + this._id;\r\n    });\r\n\r\n\r\n\r\n\r\n//Export model\r\nmodule.exports = mongoose.model('Song', SongSchema);
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/express-bandbase/models/song.js b/express-bandbase/models/song.js
--- a/express-bandbase/models/song.js	(revision ae5d4dd6ca98ea6ae7bfd046b6e81ff6f73a702b)
+++ b/express-bandbase/models/song.js	(date 1638209533288)
@@ -27,7 +27,5 @@
     });
 
 
-
-
 //Export model
 module.exports = mongoose.model('Song', SongSchema);
\ No newline at end of file
